tests/nlu/featurizers/test_count_vectors_featurizer.py

```diff
@@ -397,3 +404,102 @@ def test_count_vectors_featurizer_train():
     assert sen_vec is None
     assert (1, 1) == seq_vec.shape
     assert np.all(seq_vec.toarray()[0] == np.array([1]))
+
+
+@pytest.mark.parametrize(
+    "sentence, action_name, action_text, action_name_features, response_features",
+    [
+        ("hello", "greet", None, [[1]], None),
+        ("hello", "greet", "hi", [[1]], [[1]]),
+        ("hello", "", "hi", None, [[1]]),
+    ],
+)
+def test_count_vector_featurizer_action_attribute_featurization(
+    sentence, action_name, action_text, action_name_features, response_features
```

Comment:
types please

Reply:
same for the others

Reply:
None of the featurizer tests have the types; should I add types in all of them? or do it in a separate PR later?

Reply:
Depends on your time :-) For now I'd just add them for the ones you created

Reply:
Done!